src/main/java/com/twilio/sdk/resource/InstanceResource.java

```diff
@@ -192,4 +204,23 @@ protected Date parseDate(final String inDate) {
 		}
 	}
 
+	/**
+	 * Given a GMT date string in the format yyyy-MM-dd, return a parsed {@link java.util.Date}
+	 *
+	 * @param inDate the date String to parse
+	 * @return the parsed {@link java.util.Date} object
+	 */
+	protected Date parseSimpleDate(String inDate) {
```

Comment:
I'm not a fan of this pattern. This is leading down a path where we add more and more methods that are very similar to this.

Instead, can we add a ```parseDateWithFormat(String inDate, FastDateFormat format)``` and then call it with ```parseDateWithFormat(inDate, DateFormatUtils.ISO_DATE_FORMAT)```